test/integration/request-flow/intermediate-requests.test.js

```diff
@@ -76,9 +76,11 @@ describe('intermediate requests from auth', function () {
 
         it('must have sent the original request', function () {
             var err = testrun.response.firstCall.args[0],
+                cursor = testrun.response.firstCall.args[1],
                 request = testrun.response.firstCall.args[3];
 
             expect(err).to.be(null);
+            expect(cursor).to.not.be(null);
```

Comment:
Would it be possible to write a stricter assertion?

Reply:
A stricter assertion would be to check the shape of cursor like `expect(testrun.response.firstCall.args[1]).to.have.property('ref');`. Asserting on values is not feasible.